packages/dialtone-vue2/components/rich_text_editor/mention_suggestion.js

```diff
@@ -1,26 +1,53 @@
 /* eslint-disable max-len */
+const CONTACT_LIST = [
+  {
+    id: 'test.person',
+    name: 'Test Person',
+    avatarSrc: 'https://avatars.githubusercontent.com/u/13851061?s=460&u=1f1b5b0b5b2b2b2b2b2b2b2b2b2b2b2b2b2b2b&v=4',
+  },
+  {
+    id: 'test.person2',
+    name: 'Test Person 2',
+    avatarSrc: 'https://avatars.githubusercontent.com/u/13851061?s=460&u=1f1b5b0b5b2b2b2b2b2b2b2b2b2b2b2b2b2b2b&v=4',
+  },
+  {
+    id: 'test.person3',
+    name: 'Test Person 3',
+    avatarSrc: 'https://avatars.githubusercontent.com/u/13851061?s=460&u=1f1b5b0b5b2b2b2b2b2b2b2b2b2b2b2b2b2b2b&v=4',
+  },
+  {
+    id: 'brad.paugh',
+    name: 'Brad Paugh',
+    avatarSrc: 'https://avatars.githubusercontent.com/u/13851061?s=460&u=1f1b5b0b5b2b2b2b2b2b2b2b2b2b2b2b2b2b2b&v=4',
+  },
+  {
+    id: 'bradley.hawkins',
+    name: 'Bradley Hawkins',
+    avatarSrc: 'https://avatars.githubusercontent.com/u/13851061?s=460&u=1f1b5b0b5b2b2b2b2b2b2b2b2b2b2b2b2b2b2b&v=4',
+  },
+  {
+    id: 'julio.ortega',
+    name: 'Tico Ortega',
+    avatarSrc: 'https://avatars.githubusercontent.com/u/13851061?s=460&u=1f1b5b0b5b2b2b2b2b2b2b2b2b2b2b2b2b2b2b&v=4',
+  },
+  {
+    id: 'ignacio.Ropolo',
```

Comment:
```suggestion
    id: 'ignacio.ropolo',
```

Reply:
lol I swear I already fixed this... must have not committed it or something.